cupyx/scipy/linalg/matfuncs.py

```diff
@@ -0,0 +1,15 @@
+from numpy import exp
+
+
+def sinhm(arr):
+    """Returns hyperbolic sine of the given square matrix.
+
+Args:.
```

Comment:
Please look other cupy functions docstring and use the same format:

https://github.com/cupy/cupy/blob/master/cupyx/scipy/linalg/decomp_lu.py#L13-L35

Reply:
I will make the necessary changes.